test/micro/org/openjdk/bench/java/lang/StringBuilders.java

```diff
@@ -205,6 +205,21 @@ public String toStringCharWithString16() {
         return result.toString();
     }
 
+    @Param({"128", "256", "1024"})
```

Comment:
I think adding this parameter at this level means all other benchmarks will now be redundantly run once per different value for MIXED_SIZE. You can avoid this by refactor this parameter and the associated Benchmark into subclass.